templates/zerver/integrations.html

```diff
@@ -969,6 +969,76 @@
 
     </div>
 
+    <div id="gcal" class="integration-instructions">
+
+        <p>Get Google Calendar reminders in Zulip! This is a great way to see your reminders directly in
+        your Zulip feed.</p>
+
```

Comment:
I think we need a little more documentation here. It seems like there are two aspects to this: the server administrator needs to generate a `client_secret.json` file and store it in the appropriate location; and the bot needs to be run for any user who wants reminders. This second part is well-documented here, but it'd be good to have some documentation explaining what `client_secret.json` is, how to generate it and where that file needs to be placed.